readme.md

```diff
@@ -1,12 +1,35 @@
-![Tridactyl logo](src/static/logo/Tridactyl_200px.png)
-
-# Tridactyl [![Build Status](https://travis-ci.org/tridactyl/tridactyl.svg?branch=master)](https://travis-ci.org/tridactyl/tridactyl) [![Matrix Chat][matrix-badge]][matrix-link] [![Gitter Chat][gitter-badge]][gitter-link]
-
-Replace Firefox's default control mechanism with one modelled on the one true editor, Vim.
-
-![Gigantic GIF showing Tridactyl in action](doc/AMO_screenshots/trishowcase.gif)
-
-## Installing
+<h1 align="center">
+<br>
+<img src="src/static/logo/Tridactyl_200px.png" alt="Tridactyl Logo" width="200">
+<br>
+Tridactyl
+<br>
+</h1>
+
+<h4 align="center">Replace Firefox's default control mechanism with one modelled on the one true editor, Vim. </h4>
+
+<p align="center">
+<a href="https://travis-ci.org/tridactyl/tridactyl"><img src="https://travis-ci.org/tridactyl/tridactyl.svg?branch=master" alt="Build Status"></a>
+<a href="https://gitter.im/tridactyl/Lobby"><img src="https://badges.gitter.im/Join%20Chat.svg" alt="Join Gitter Chat"></a>
+<a href="https://addons.mozilla.org/en-US/firefox/addon/tridactyl-vim/"><img src="https://img.shields.io/amo/users/tridactyl-vim" alt="Mozilla Addon Store"></a>
```

Comment:
This is a neat idea, but it undercounts our users quite significantly: the AMO only counts people who have opted in to sharing usage data (when measured a different way by Mozilla we had approx 7k DAU). It also excludes beta users (~600 in the new way so ~1k in the old way) and Arch users (of which we have god knows https://pkgstats.archlinux.de/packages/firefox-tridactyl ).

Generally I tell people that we have about 10k weekly users. That's essentially my concern: I don't want people who I've told that to to think I'm exaggerating, so I'm not sure I want it displayed so prominently without context : )